docs/advanced-topics/checksum-support.rst

```diff
@@ -94,15 +94,15 @@ or `Azure <https://docs.microsoft.com/en-us/azure/container-registry/container-r
 Tools
 -----
 As noted in the table above, Docker image checksums are grabbed on refresh and should work as long as the image is from Quay.io, Docker Hub,
-or GitLab. It's also important to note that this is done for the Docker image registered on the tool through Dockstore and not necessarily
+Amazon ECR, or GitLab. It's also important to note that this is done for the Docker image registered on the tool through Dockstore and not necessarily
```

Comment:
It was also like this, but I think "the one included within the descriptor file itself" should be "the one **referenced** within the descriptor file itself"